tests/test_clientside/clientside.js

```diff
@@ -0,0 +1,27 @@
+window.clientside = {
+
+    display: function (value) {
+        return 'Client says "' + value + '"';
+    },
+
+    mean: function (...args) {
+        return R.mean(args);
+    },
+
+    add1_break_at_11: function (value) {
+        if (parseInt(value, 10) === 11) {
+            throw new Error('Unexpected error');
+        }
+        return parseInt(value, 10) + 1;
+    },
+
+    add_to_four_outputs: function(value) {
+        return [
+            parseInt(value) + 1,
+            parseInt(value) + 2,
+            parseInt(value) + 3,
+            parseInt(value) + 4
+        ]
+    }
+
+}
```

Comment:
As https://github.com/plotly/dash-renderer/pull/143/files#diff-4e3037116dbc2bc69a473b7ace3d02abR508 only supports sync callbacks, would like to see a negative test handling a promise case & failure.